pyproject.toml

```diff
@@ -17,8 +18,9 @@ testpaths = [
 
 # mypy global options:
 [tool.mypy]
-python_version = "3.8"
-check_untyped_defs = true
```

Comment:
Any reason this was removed?

Reply:
the `MUICore_pb2_grpc.py` file was raising some errors caused by some "experimental api" code that isn't actually used anymore but is still generated, but I've restored these 2 options and now excluded that file